package.json

```diff
@@ -12,14 +12,15 @@
     "dev": "NODE_ENV=development webpack-dev-server --hot --progress --https",
     "dev:auth": "NODE_ENV=development SDK_ENV=Auth webpack-dev-server --hot --progress --https",
     "travis": "node_modules/http-server/bin/http-server dist/ -S -C test/cert.pem -K test/key.pem",
+    "prebuild": "npm run clean",
     "build": "webpack --quiet --display-error-details --bail",
     "build:auth": "SDK_ENV=Auth webpack --quiet --display-error-details --bail",
     "build:dev": "NODE_ENV=development npm run build",
     "build:test": "NODE_ENV=test npm run build && npm run mock-server:build",
     "build:all": "npm run build && npm run build:auth",
     "postbuild": "dts-bundle-generator src/types/index.ts --out-file lib/index.d.ts && npm run license-checker",
     "clean": "rimraf lib dist",
-    "prepare": "npm run clean && npm run check && npm run build:all",
+    "prepublish": "npm run build:all",
```

Comment:
This script has been deprecated according to the npm docs https://docs.npmjs.com/cli/v7/using-npm/scripts#prepare-and-prepublish I think the script to run is `prepublishOnly` depending on the use case.